app/assets/javascripts/single_page/dynamic_table.js.erb

```diff
@@ -20,9 +20,9 @@ const defaultCols = [{
   }
 }];
 
-const objectInputTemp = '<input type="hidden" name="_NAME_[]" id="h_NAME_" value="" autocomplete="off" />' +
+const objectInputTemp = '<input type="hidden" name="_NAME_[]" id="_NAME_" value="" autocomplete="off" />' +
   '<select name="_NAME_[]" id="_NAME_" class="form-control" data-role="seek-objectsinput" ' +
```

Comment:
The `input` element should have a different `id` to the `select` tag (or just get rid of the id altogether if you don't need to lookup the element later)

Reply:
I thought it was a typo I overlooked. I changed it to this to make it clear:

```html
<input type="hidden" name="56-57-58-54-2e18bf04-6593-4c3f-9e41-994a1b8eec0b[]" id="inpt-56-57-58-54-2e18bf04-6593-4c3f-9e41-994a1b8eec0b" value="" autocomplete="off">
<select name="56-57-58-54-2e18bf04-6593-4c3f-9e41-994a1b8eec0b[]" id="select-56-57-58-54-2e18bf04-6593-4c3f-9e41-994a1b8eec0b" class="form-control select2-hidden-accessible" data-role="seek-objectsinput" data-tags-limit="100" multiple="" data-typeahead-template="typeahead/single_pages_samples" data-typeahead-query-url="/samples/typeahead?linked_sample_type_id=16" tabindex="-1" aria-hidden="true">
  <option selected="selected" title="ID: 56" value="56">Sample C</option>
  <option selected="selected" title="ID: 57" value="57">Sample D</option>
  <option selected="selected" title="ID: 58" value="58">Sample E</option>
  <option selected="selected" title="ID: 54" value="54">Sample A</option>
</select>
```